modules/installation/proc-rhdh-deploy-eks-helm.adoc

```diff
@@ -113,58 +105,22 @@ upstream:
   volumePermissions:
     enabled: true
 ----
+--
 
-. Run the following command in your terminal to deploy {product-short} using the latest version of Helm Chart and using the `values.yaml` file created in the previous step:
+. Run the following command in your terminal to deploy {product-short} using the latest version of Helm Chart and using the values.yaml file created in the previous step:
```

Comment:
Not sure why this change was made - should the monospace formatting be reapplied?